code/game/machinery/portable_turret.dm

```diff
@@ -13,6 +13,9 @@
 	icon_state = "turretCover"
 	anchored = 1
 
+/obj/machinery/porta_turret
+	name = "one star turret"
+
```

Comment:
I think you meant `/obj/machinery/porta_turret/One_star`. And the location of these line seems off, it is in the middle of another declaration.

Reply:
my mistake, fixed, how byond pass this without compile errors

Reply:
https://github.com/discordia-space/CEV-Eris/pull/4237/commits/4b03ab778678ce13e1af4b6f0cb8d388fe580481